src/main/java/com/dabsquared/gitlabjenkins/webhook/status/StatusJsonAction.java

```diff
@@ -24,13 +25,24 @@ public StatusJsonAction(Job<?, ?> project, String sha1) {
 
     @Override
     protected void writeStatusBody(StaplerResponse response, Run<?, ?> build, BuildStatus status) {
+        GitLabPushTrigger trigger = null;
```

Comment:
The `StatusJsonAction` is only required for GitLab 7.x. As the latest version of the plugin dropped official support for this GitLab version this needn't to be implemented.